ajax/libs/numbro/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "numbro",
+  "version": "1.1.1",
+  "description": "Format and manipulate numbers.",
+  "homepage": "http://numbrojs.com",
+  "author": {
+    "name": "Företagsplatsen AB",
+    "email": "contact@foretagsplatsen.se",
+    "url": "http://www.foretagsplatsen.se"
+  },
+  "keywords": [
+    "numeral",
+    "numbro",
+    "number",
+    "format",
+    "time",
+    "money",
+    "percentage"
+  ],
+  "main": "./numbro.js",
+  "engines": {
+    "node": "*"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/foretagsplatsen/numbro"
+  },
+  "bugs": {
+    "url": "https://github.com/foretagsplatsen/numbro/issues"
+  },
+  "licenses": [
+    {
+      "type": "MIT"
+    }
+  ],
+  "devDependencies": {
+    "grunt": "latest",
+    "grunt-bump": "latest",
+    "grunt-confirm": "latest",
+    "grunt-contrib-concat": "latest",
+    "grunt-contrib-jshint": "latest",
+    "grunt-contrib-nodeunit": "latest",
+    "grunt-contrib-uglify": "latest",
+    "grunt-release": "latest",
+    "uglify-js": "latest"
+  },
+  "scripts": {
+    "test": "grunt test"
+  },
+  "npmName": "numbro",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "*"
```

Comment:
Then I will say I used `cdnjs-importer` as explained in the doc, so I have no clue :smile